crud/select/merger.lua

```diff
@@ -80,6 +70,23 @@ local function decode_response_headers(buf)
     return ffi.cast(char_ptr, data[0])
 end
 
+local function decode_response_headers(buf)
+    data[0] = buf.rpos
+
+    if not utils.tarantool_supports_netbox_skip_header_option() then
+        -- {48: [cursor, [tuple_1, tuple_2, ...]]} (exactly 1 pair of key-value)
+
+        -- 48 (key)
+        data[0] = data[0] + 1
+
+        --{48: [cursor, [tuple_1, tuple_2, ...]], 96: {id: tuple_format}}
```

Comment:
Or this comment is not on its place or I miss something. This branch is where we can't see `IPROTO_TUPLE_FORMATS`, correct?

Reply:
That's a typo, sorry.